engine/schema/src/main/java/com/cloud/vm/VmStatsVO.java

```diff
@@ -0,0 +1,80 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.cloud.vm;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Entity
+@Table(name = "vm_stats")
+public class VmStatsVO {
```

Comment:
can you also add instancename(i-) and the display name also so that it will be very easy to find out the vm name else we need to query on id again to get the vm name?

Reply:
@ravening 
I didn't quite understand your suggestion. This class represents an entity in the database that is not the entity of VMs. If I add the intancename, every time the user changes the VM name it would have to be changed here too (for each VM stats record), which ends up adding unnecessary complexity/processing.

If you check the output samples of the API (in the screenshots added in PR), it returns the name and display name of the VM.

Reply:
@joseflauzino i- name is fixed and users cant change it. from the logs we usually get i- name and so if this table has the i- name then searching the entries using i- name is easy

Reply:
@ravening 
I think that instead of replicating the `instance_name` field in different tables, we can just use JOIN.
To get stats from a specific `instance_name` we can use something like:
```SQL
SELECT vm.instance_name, vs.timestamp, vs.vm_stats_data
FROM vm_stats vs
JOIN vm_instance vm
ON vs.vm_id=vm.id
WHERE vm.instance_name = ?
```
Therefore, I do not see the reason to add the `instance_name` in the `vm_stats` table.